packages/firebase/index.d.ts

```diff
@@ -1,3 +1,5 @@
+import { FieldPath } from '../firestore-types/index';
```

Comment:
This should probably go below the copyright?

Reply:
This is actually not needed since FieldPath is defined in this very file.